src/store/flow/utils/importHelpers.ts

```diff
@@ -27,21 +27,26 @@ export function updateResourcesForLanguageMatch(
 export function mergeFlowContainer(
   existingFlowContainer: IContext, newFlowContainer: IContext,
 ): IContext {
-  const newExistingFlowContainer = cloneDeep(existingFlowContainer)
-  // There should only be a single flow imported at a time currently.
-  // We block any more than this in the importer
-  const newFlow = get(newFlowContainer, 'flows[0]')
-  const newFlowUUID = get(newFlowContainer, 'flows[0].uuid')
-  const existingFlowIndex = findIndex(
-    newExistingFlowContainer.flows,
-    (flow) => flow.uuid === newFlowUUID,
-  )
-  if (existingFlowIndex < 0) {
-    newExistingFlowContainer.flows.push(newFlow)
-  } else {
-    newExistingFlowContainer.flows[existingFlowIndex] = newFlow
-  }
-  return newExistingFlowContainer
+  const existingFlowContainerStore = cloneDeep(existingFlowContainer);
```

Comment:
why did we change `newExistingFlowContainer` into `existingFlowContainerStore`? 
- let's avoid using `store` prefix as this brings confusion to vuex stores, we have a lot of that in this repo
- unless you have a strong opinion on the renaming, I'd recommend we keep `newExistingFlowContainer` or use `clonedExistingFlowContainer`

- [ ] improve the variable name